lib/matplotlib/figure.pyi

```diff
@@ -1,3 +1,7 @@
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
+    from mpl_toolkits.mplot3d import Axes3D
+
```

Comment:
`typing` is already imported on line 7, so this should move there. And `mpl_toolkits` should go in the third block with the rest of Matplotlib.

Reply:
Hrm, typing is already used on two other lines, so I'll unify that too then.

Reply:
Fixed!